tests/testthat/test_base_performance.R

```diff
@@ -41,7 +41,7 @@ test_that("performance", {
   # custom measure
   mymeasure = makeCustomResampledMeasure(measure.id = "mym", aggregation.id = "train.mean",
     properties = c("classif", "predtype.response"),
-    fun = function(task, group, pred, feats, extra.args) {
+    fun = function(task, model, pred, feats, extra.args) {
```

Comment:
What does this change do? Looks like it will affect other tests as well?

Reply:
I think this was just wrong in the former version?

Reply:
Maybe its not used in the test and was not catched therefore. If you're sure that its good now than its fine :)